### PR TITLE
SEO-171754 - wpf controls-2

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @Aishwarya-Ganesan ,

I have made the changes in redirect links. Please review.

Thanks.

Regards,
Edith.

### CODE REVIEW

File: wpf/DataGrid/Column-Types.md [hunk 1]
```diff
@@ -2758,7 +2758,7 @@ public class GridCellTextBoxRendererExt:GridCellTextBoxRenderer
 
 ### Create the renderer of existing column
 
-You can change the renderer of existing column by removing the predefined cell type value from [CellRenderers](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CellRenderers) collection and add the newly derived renderer from [GridVirtualizingCellRenderer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer%602.html).
+You can change the renderer of existing column by removing the predefined cell type value from [CellRenderers](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CellRenderers) collection and add the newly derived renderer from [GridVirtualizingCellRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer-2.html).
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Column-Types.md:2761]: Recheck the above link and check all the links

File: wpf/DataGrid/Column-Types.md [hunk 2]
```diff
@@ -3015,7 +3015,7 @@ public class DisplayConverter : IValueConverter
 
 ### Creating new column and renderer
 
-You can create a new column by deriving [GridColumn](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html), rendered in UI using customized [CellType](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html#Syncfusion_UI_Xaml_Grid_GridColumn_CellType) using [GridVirtualizingCellRenderer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer%602.html).
+You can create a new column by deriving [GridColumn](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html), rendered in UI using customized [CellType](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html#Syncfusion_UI_Xaml_Grid_GridColumn_CellType) using [GridVirtualizingCellRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.CellGrid.GridCellRenderer.GridVirtualizingCellRenderer-2.html).
```
Comment (Yvone-Atieno) [inline wpf/DataGrid/Column-Types.md:3018]: Update proper link

Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md:3018]: Have checked and all the changed links here are correct, kindly confirm this from your end also.

File: wpf/DataGrid/Column-Types.md (hunk 1 above)
Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md:2761]: Have rechecked the link is correctly updated here.

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**